__tests__/app.test.js

```diff
@@ -1,7 +1,20 @@
+// contains static data
 const endpointsJson = require("../endpoints.json");
-/* Set up your test imports here */
+// allows endpoint connections, supertest provides its own listeners when running tests
+const request = require("supertest");
+const app = require("../app");
+// sample data provided for testing and seed to be populated with such data
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+// database connection pool for ending connection after tests run
+const db = require("../db/connection");
```

Comment:
do you need all these comments?

Reply:
You've a point, I'll remove those comments for the next commit and stick to populating the README when necessary